src/airflow_declarative/trafaret.py

```diff
@@ -188,6 +189,23 @@ def cast_interval(value):
     return value
 
 
+def cast_crontab_or_interval(value):
+    try:
+        return cast_interval(value)
+    except ValueError:
+        if not isinstance(value, str):
+            raise
+        # This is definitely not a valid time interval.
+        # Perhaps this is a cron expression?
+        try:
+            # Airflow uses `croniter` as well.
+            croniter(value)
+        except ValueError as e:
+            raise t.DataError("Invalid crontab expression: %s" % str(e))
```

Comment:
In case when both fails it's better to tell that this value is not an interval value nor crontab expression. Otherwise it will cause confusion: I want interval `2.5d`, but never asked about crontab.

Reply:
Good point, changed the error message.